### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by sidagrawal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Crypto-classical
```
Comment (fosskers) [inline README.md]:
```suggestion
# `crypto-classical`
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+An educational tool for studying classical cryptography schemes. Do not encrypt anything of worth with this library.
+
+Included Ciphers:
+ - Caesar
+ -  Affine (Linear)
+ - Substitution
+ -  Stream
```
Comment (fosskers) [inline README.md]: A few of these aren't aligned properly.

**Review by fosskers — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sidagrawal) [reply README.md]: Changed it

File: README.md (hunk 2 above)
Comment (sidagrawal) [reply README.md]: Don't know how this happened but I aligned them.

**Review by fosskers — APPROVED** [review]

**Comment by fosskers** [comment]
Thanks bud!

**Pull request merged**